app/services/katello/pulp3/repository_mirror.rb

```diff
@@ -87,9 +93,8 @@ def remote_options
         base_options = common_remote_options
         if (type_specific_options = repo_service.try(:mirror_remote_options))
           base_options.merge(type_specific_options)
-        else
-          base_options.merge(url: remote_feed_url)
         end
+        base_options.merge(url: remote_feed_url)
       end
```

Comment:
i think this would break the docker syncing which is supplying its own 'url', which gets overridden here.  You might wanna move this above the if statement